deploy/role_ui.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: noobaa-ui
```

Comment:
maybe change to `noobaa-odf-ui`, just to be more clear? I'm not really sure what is a good name here

Reply:
✅